css/style.css

```diff
@@ -0,0 +1,70 @@
+*{
+    margin:0
+  
+}
```

Comment:
Well done on eliminating margins👍🏼
Don't forget to leave a space between "margin:" and "0" for best readability.